student.rb

```diff
@@ -1,12 +1,20 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  attr_accessor :classroom
+
```

Comment:
- [Optional] Its good practice to have the accessor calls at the top of the classes and not inside. Kindly consider this 👍